### ISSUE TITLE
[Feature Request] Price calculation Car power consumption

### ISSUE THREAD

**Issue by ronzle** [solectrus/solectrus#1513]
First of all, very nice work, thanks for that !
The Senec app isn't very helpful, so your project is a huge benefit!

A nice feature would be, to add a cost calculation based on the power working costs for the power consumption of the car and put it on the overview pages as a dedicated circle. 

/ronny

**Comment by ledermann** [comment]
Yes, that sounds interesting! Calculating the cost of charging the car in a certain time frame (day, week, month, year, total) would be a nice addition. I will think about it, thanks for this suggestion, @ronzle!

**Comment by ronzle** [comment]
Hi 

do you plan a docker image as preview for 0.8 ? 
i could do some tests for you and give some feedback 

cheers 
Ronny

**Comment by ledermann** [comment]
@ronzle Yes, there is a Docker image with this feature:
`ghcr.io/solectrus/solectrus:pr-1548`

But be warned: The implementation is still very rough. Both the calculation and the user interface will most likely change.

**Comment by ledermann** [comment]
Recently, I improved and corrected the calculation and merged the code into `develop`, so the Docker image mentioned above no longer exists. You can use this for testing:
`ghcr.io/solectrus/solectrus:develop`

<img width="655" alt="CleanShot 2023-03-07 at 10 04 42@2x" src="https://user-images.githubusercontent.com/3557/223374805-b9070c8d-d571-4dd9-aa50-d200cfaafa26.png">

**Comment by ledermann** [comment]
Today released v0.9.0 including this feature.
